x/ethbridge/client/rest/rest.go

```diff
@@ -29,6 +29,7 @@ type createEthClaimReq struct {
 	CosmosReceiver string       `json:"cosmos_receiver"`
 	Validator      string       `json:"validator"`
 	Amount         string       `json:"amount"`
+	ClaimType      string       `json:"claim_type"`
 }
 
 type burnEthReq struct {
```

Comment:
`lockEthReq struct` and `func lockHandler` aren't included here. Just want to confirm that the reason they aren't needed is because MsgLock is leveraging the existing structure of create-claim?

Reply:
Nope, i just forgot it. Will add.

Reply:
https://github.com/cosmos/peggy/pull/81/commits/265f143dabbe9debd23952df7c4130ec4ed1e916